app/assets/stylesheets/provider/_theme.scss

```diff
@@ -1,72 +1,70 @@
-@import 'provider/typography';
+@forward 'provider/typography';
```

Comment:
I guess `@forward` is appropriate in this file:
https://sass-lang.com/documentation/at-rules/forward/